src/pages/Index/Section/Personal/index.js

```diff
@@ -36,16 +63,30 @@ const Header = ({ tabState }) => (
 )
 
 const Personal = () => {
-  const tabState = useState(loadTab)
-  const activeTab = tabState[0]
+  const [activeTab, setTab] = useState(loadTab)
   const Content = TabTypeComponent[activeTab]
 
+  const history = useHistory()
```

Comment:
Stylistically better to group hooks together, i.e. `useHistory()` could be placed after `const tabState = useState(loadTab)`